Makefile

```diff
@@ -40,9 +40,12 @@ BUILDTYPE_LOWER := $(shell echo $(BUILDTYPE) | tr '[A-Z]' '[a-z]')
 EXEEXT := $(shell $(PYTHON) -c \
 		"import sys; print('.exe' if sys.platform == 'win32' else '')")
 
-NODE_EXE = node$(EXEEXT)
+# Build to /node for easy cross-repo compat.
+BUILD_DIR_NAME = node
```

Comment:
I have to admit I don’t quite get why this is called `DIR_NAME` and why `$(EXEEXT)` is removed? I mean, it’s still the executable name in the build dir, right…